rampwf/tests/kits/titanic_test_external_imports/external_imports/utils/test_imports.py

```diff
@@ -0,0 +1 @@
+x = 2
```

Comment:
also maybe add a small comment here to say that this is just to test external imports